Documentation/gettingstarted/http.rst

```diff
@@ -364,6 +369,15 @@ and ``cilium`` CLI:
     ]
     Revision: 11
 
+It is also possible to monitor the HTTP requests live by using ``cilium monitor``:
+
+::
+
+    $ kubectl exec -it -n kube-system cilium-kzgdx -- cilium monitor -v --type l7
+    <- Response http to 0 ([k8s:class=tiefighter k8s:io.cilium.k8s.policy.cluster=default k8s:io.cilium.k8s.policy.serviceaccount=default k8s:io.kubernetes.pod.namespace=default k8s:org=empire]) from 2756 ([k8s:io.cilium.k8s.policy.cluster=default k8s:class=deathstar k8s:org=empire k8s:io.kubernetes.pod.namespace=default k8s:io.cilium.k8s.policy.serviceaccount=default]), identity 8876->43854, verdict Forwarded POST http://deathstar.default.svc.cluster.local/v1/request-landing => 200
+    <- Request http from 0 ([k8s:class=tiefighter k8s:io.cilium.k8s.policy.cluster=default k8s:io.cilium.k8s.policy.serviceaccount=default k8s:io.kubernetes.pod.namespace=default k8s:org=empire]) to 2756 ([k8s:io.cilium.k8s.policy.cluster=default k8s:class=deathstar k8s:org=empire k8s:io.kubernetes.pod.namespace=default k8s:io.cilium.k8s.policy.serviceaccount=default]), identity 8876->43854, verdict Denied PUT http://deathstar.default.svc.cluster.local/v1/request-landing => 403
+
+You can observe how, as expected, the first POST request succeeded while the second PUT request was denied
```

Comment:
nit, grammar:

```suggestion
You can observe how, as expected, the first POST request succeeded while the second PUT request was denied.
```

Though to be more precise, the monitor output shows only a successful POST *response* followed by a PUT request that is denied. 

I won't enforce this since the guide is already written this way, but in general I would prefer to use neutral, factual language rather than language like "You can observe". An example would be something like "The above output demonstrates a successful response to a POST request followed by a PUT request that is denied by the L7 policy". This is a more direct description of the output rather than telling the user what they may be able to observe (assuming they understand what they're looking at).